lib/Dialect/TTIR/IR/TTIROpsInterfaces.cpp

```diff
@@ -5,12 +5,44 @@
 #include "ttmlir/Dialect/TTIR/IR/TTIROpsInterfaces.h"
 #include "ttmlir/Dialect/TTIR/IR/TTIR.h"
 
+#include <cstdint>
+#include <llvm/ADT/SmallVector.h>
+#include <mlir/Dialect/Traits.h>
+#include <mlir/IR/BuiltinAttributes.h>
+#include <mlir/IR/Operation.h>
 #include <mlir/Support/LogicalResult.h>
 
 mlir::LogicalResult
 mlir::tt::ttir::detail::verifyElementwiseOp(mlir::Operation *op) {
-  // Currently, elementwise trait already performs the basic verification.
-  // Let this be a placeholder for future extensions.
+  llvm::SmallVector<int64_t> broadcastedShape;
```

Comment:
Micro-opt, but you can define the small vector to take 4 elements if that is the most common shape we expect to encounter.